hazelcast-client-new/src/test/java/com/hazelcast/client/ClientConnectionTest.java

```diff
@@ -42,36 +41,37 @@
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(HazelcastSerialClassRunner.class)
+@RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientConnectionTest extends HazelcastTestSupport {
 
+    private final TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
```

Comment:
@pveentjer https://github.com/hazelcast/hazelcast/issues/5740 
Feel free to edit the explanation of ticket, if you think it needs more clarification.